apps/communications/contacts/views/details/js/boot.js

```diff
@@ -0,0 +1,58 @@
+/* global DetailsUI, Details, LazyLoader, fbLoader, ContactsService */
+'use strict';
+
+window.addEventListener('DOMContentLoaded', function() {
+  LazyLoader.load([
+    '/shared/js/l10n.js'], function() {
+    LazyLoader.load([
+      document.getElementById('view-contact-details')
+    ], function() {
+      // TODO Add if needed
+    });
+  });
+});
+
+window.onload = function() {
+  var dependencies = [
+    '/contacts/style/webrtc-client/webrtc_client.css',
+    '/contacts/js/webrtc-client/webrtc_client.js',
+    '/contacts/services/contacts.js',
+    '/contacts/js/activities.js',
+    '/shared/js/contacts/utilities/event_listeners.js',
+    '/shared/js/l10n_date.js',
+    '/shared/js/contacts/import/utilities/config.js',
```

Comment:
Cannot remember why we needed the config, cause of the image configuration parameters?